gcn_classic_to_json/notices/GECAM_FLT/__init__.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+from ..GECAM_GND import parse as parse_gecam_gnd
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[27:39]  # Unused. According to docs: '32 bytes for the future'
+    bin[21]  # Temporarily Unused. This is trigger_duration; currently no keyword
```

Comment:
Please work with @Vidushi-GitHub to create a schema entry for this field.

Reply:
My bad, I should have removed this line! After speaking with @Vidushi-GitHub, we decided to name this trigger_duration i.e. keep the name from the source code which is present in the return statement.